app/lib/azureService.js

```diff
@@ -0,0 +1,32 @@
+const azure = require('azure-storage');
+
+const config = require('./config.js');
+
+const blobSvc = azure.createBlobService();
+
+function listBlobs() {
+  return new Promise((resolve, reject) => {
+    blobSvc.listBlobsSegmented(config.containerName, null, (error, result) => {
+      if (!error) {
+        // result.entries contains the entries
+        // If not all blobs were returned, result.continuationToken has the continuation token.
+        resolve(result.entries);
+      }
+      reject(error);
+    });
+  });
+}
+
+function uploadToAzure(filePath, name) {
+  return new Promise((resolve, reject) => {
+    blobSvc.createBlockBlobFromLocalFile(config.containerName, name, filePath, (error, result) => {
```

Comment:
Not sure we allow permissions to create containers, or want to have containers created by applications, might be a question for Adam.

If there is no container the promise will get rejected with the error 'no container found'